templates/base.html

```diff
@@ -51,6 +51,15 @@
     <script src="{% static "components/lodash/dist/lodash.min.js" %}"></script>
     <script src="{% static "components/moment/min/moment.min.js" %}"></script>
     <script src="{% static "components/remarkable-bootstrap-notify/dist/bootstrap-notify.min.js" %}"></script>
+     <!-- <script src="https://code.jquery.com/jquery-1.11.3.min.js"></script> -->
+  <script src="http://js.cytoscape.org/js/cytoscape.min.js"></script>
+  <!-- <script src="https://github.com/tgdwyer/WebCola/blob/master/WebCola/cola.min.js"></script> -->
+  <!-- <script src="https://marvl.infotech.monash.edu/webcola/cola.v3.min.js"></script> -->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+  <!-- <script src="https://cdnjs.cloudflare.com/ajax/libs/cytoscape/2.7.16/cytoscape.min.js"></script> -->
+  <script src="https://bioinformatics.cs.vt.edu/~jeffl/cola.min.js"></script>
```

Comment:
Why are we loading from jeffs webserver. Can you load from bower package instead?